Content.Server/Store/Systems/StoreSystem.Ui.cs

```diff
@@ -165,8 +165,6 @@ private void OnBuyRequest(EntityUid uid, StoreComponent component, StoreBuyListi
 
         if (!IsOnStartingMap(uid, component))
             component.RefundAllowed = false;
```

Comment:
could you not just wrap this in a check for whether or not the store is able to refund? or is setting it to true just not needed at all.

Reply:
Wizard should be able to refund before they leave their ship. If we ever get a cult it'd be up for debate if they should be able to buy/refund powers offstation.